content/source/docs/enterprise/install/automating-the-installer.html.md

```diff
@@ -180,9 +180,10 @@ If you have chosen `production` for the `installation_type`, `production_type` i
 
 - `production_type` — One of `external` or `disk`.
 
-If you have chosen `disk` for `production_type`, `disk_path` is required:
+If you have chosen `disk` for `production_type`, `disk_path` and `pg_password` are required:
```

Comment:
The `pg_password` setting is not currently required. One will be generated if not provided. However, I do agree that it should be documented and that we should recommend that the user explicitly set this value so that it is known for re-installations. I do not believe that we should call it required until it actually becomes, if ever, required in the settings.

Reply:
Updated, thank you!